src/components/va-alert/va-alert.tsx

```diff
@@ -127,10 +127,8 @@ export class VaAlert {
   }
 
   render() {
-    const { backgroundOnly, status, fullWidth, visible, closeable } = this;
-    const classes = `alert ${status} ${fullWidth ? 'full-width' : ''} ${
-      backgroundOnly ? 'bg-only' : ''
-    }`;
+    const { backgroundOnly, status, visible, closeable } = this;
+    const classes = `alert ${status} ${backgroundOnly ? 'bg-only' : ''}`;
```

Comment:
This is what we had before the changes in #181:

https://github.com/department-of-veterans-affairs/component-library/blob/3694481bca2490adc534effed971bd53dc1806dc/src/components/va-alert/va-alert.tsx#L124-L125